extensions/3DTILES_layers/README.md

```diff
@@ -0,0 +1,98 @@
+# 3DTILES_layers
+
+## Overview
+
+This extension to 3D Tiles enables grouping of different types of content into layers. A layer is a user-defined grouping of content - for example, a tileset of a city may have a buildings layer, a roads layer and a vegetation layer. Each layer follows an independent layout for its tiles and there are not dependencies between layers. A layer may also associate with itself some metadata by conforming to a `class` declared in `3DTILES_metadata`.
```

Comment:
```suggestion
This extension to 3D Tiles enables assigning tile content to layers.

A layer is a user-defined grouping of content - for example, a tileset of a city may have a buildings layer, a roads layer and a vegetation layer. There are no dependencies between layers. A layer may contain application-specific metadata by conforming to a `class` defined in `3DTILES_metadata`.
```

> Each layer follows an independent layout for its tiles

I removed this part because it's not super clear what it means. Could it be reworded?

Reply:
It's worth having a paragraph or two talking about what layers enable.

* Enables a tile to have multiple contents that can be requested independently from each other. Avoids the drawback of the cmpt format which is all or nothing.
* Datasets that inherently follow the same tiling scheme don't need to be stored in separate tileset.json files
* Provides a way to pair regular tile content like b3dm, glTF with application specific content, e.g. some game engine specific payload.
* Would enable future capabilities like navigation meshes, physics meshes as separate layers
* Styling use cases: show/hide, change color. Enabled automatically by the styling language since layer properties are inherited by features.
* For some applications, z-order